back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -41,8 +41,9 @@
       parameter :folder, 'Filter by folder (project folder id)', required: false
       parameter :remove_not_allowed_parents, 'Filter out folders which contain only projects that are not visible to the user', required: false
       parameter :only_projects, 'Include projects only (no folders)', required: false
-      parameter :filter_can_moderate, 'Filter out the projects the user is allowed to moderate. False by default', required: false
-      parameter :filter_is_moderator_of, 'Filter out the publications the user is not moderator of. False by default', required: false
+      parameter :filter_can_moderate, 'Filter out the projects the current_user is allowed to moderate. False by default', required: false
```

Comment:
Publications, not projects?

Reply:
Also, I find the phrase "filter out" a bit confusing. Sounds to me like whatever is filtered out is excluded from the list where the opposite is happening here.